sa/sa.go

```diff
@@ -1554,9 +1568,13 @@ func (ssa *SQLStorageAuthority) GetAuthorizations2(ctx context.Context, req *sap
 // the authorization is being moved to invalid the validationError field must be set. If the
 // authorization is being moved to valid the validationRecord and expires fields must be set.
 // This method is intended to deprecate the FinalizeAuthorization method.
-func (ssa *SQLStorageAuthority) FinalizeAuthorization2(ctx context.Context, req *sapb.FinalizeAuthorizationRequest) error {
+func (ssa *SQLStorageAuthority) FinalizeAuthorization2(ctx context.Context, req *sapb.FinalizeAuthorizationRequest) (*emptypb.Empty, error) {
+	if req.Status == "" || req.Attempted == "" || req.Expires == 0 || req.Id == 0 {
+		return &emptypb.Empty{}, errIncompleteRequest
```

Comment:
Return nil, not empty, when returning errors.

Reply:
Ooof, my bad.